lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -0,0 +1,547 @@
+package drivers
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/lxc/lxd/lxd/util"
+	"github.com/lxc/lxd/shared"
+	"github.com/lxc/lxd/shared/ioprogress"
+	"github.com/lxc/lxd/shared/logger"
+	"golang.org/x/sys/unix"
+)
+
+var errBtrfsNoQuota = fmt.Errorf("Quotas disabled on filesystem")
```

Comment:
btrfsNoQuotaErr

Reply:
This will cause static analysis to fail with `lxd/storage/drivers/driver_btrfs_utils.go:21:5: error var btrfsNoQuotaErr should have name of the form errFoo`.

Reply:
oh right, okay, fine :)
`errBtrfsXYZ` it is then as a pattern.